libbpf-tools/statsnoop.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __STATSNOOP_H
+#define __STATSNOOP_H
+
+#define TASK_COMM_LEN   16
+#define NAME_MAX        255
```

Comment:
256 for nice rounded (in binary) number?

Reply:
`linux/limits.h` has this macro which also set to 255. Maybe because it can fit in a `u8` ?

Reply:
oh, ok, it doesn't matter much, compiler will just pad one byte at the end. Please disregard, I didn't realize it's matching some Linux UAPI constant.